README.md

```diff
@@ -91,22 +92,39 @@ This release changes the way in which location data is collected to better align
 - [improvement] [cht-android#111](https://github.com/medic/cht-android/issues/111): Compliance with Play Store developer policies for PII collection disclosure
 - [bug] [cht-core#6648](https://github.com/medic/cht-core/issues/6648): Blank screen when launching external apps from CHT Android app
 
+
 # Development
 
-1. Install Android SDK
-2. Clone the repo
-3. Plug in your phone. Check it's detected with `adb devices`
-4. Execute: `make` (will also push app unto phone)
+1. Install the [Android SDK](https://developer.android.com/studio#command-tools) and Java 8+ (it works with the OpenJDK versions).
```

Comment:
Do we need to specify that we're using gradle 7?

Reply:
No because the `gradlew` script takes care of that, downloading Gradle 7.1 in the user space the first time is executed, and the build is still compatible with older versions of Gradle, but I'll add a note about it.